### PR TITLE
docs(annotations): corrections to @Query

### PR DESCRIPTION
**Pull request by eggers**
The docs still show @Children and @Descendents when it looks like that has been updated to @Query and @QueryDescendents. Also, the DirectiveType should be Dependency with the current code example.

### CODE REVIEW

**Comment by eggers** [comment]
I signed it!

File: modules/angular2/src/core/annotations_impl/annotations.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ import {DEFAULT} from 'angular2/change_detection';
  *    Shadow DOM root. Current element is not included in the resolution, therefore even if it could resolve it, it will
  *    be ignored.
  * - `@Parent() directive:DirectiveType`: any directive that matches the type on a direct parent element only.
- * - `@Children query:Query<DirectiveType>`: A live collection of direct child directives (will be implemented in later release).
- * - `@Descendants query:Query<DirectiveType>`: A live collection of any child directives (will be implemented in later relaese).
+ * - `@Query query:QueryList<DirectiveType>`: A live collection of direct child directives.
```
Comment (mhevery) [inline modules/angular2/src/core/annotations_impl/annotations.js]: This should be `@Query(DirectiveType)`

File: modules/angular2/src/core/annotations_impl/annotations.js [hunk 2]
```diff
@@ -49,8 +49,8 @@ import {DEFAULT} from 'angular2/change_detection';
...
+ * - `@QueryDescendants query:QueryList<DirectiveType>`: A live collection of any child directives.
```
Comment (mhevery) [inline modules/angular2/src/core/annotations_impl/annotations.js]: This should be `@QueryDescendants(DirectiveType)`

**Comment by eggers** [comment]
@mhevery Updated pull request for comments.

**Comment by vicb** [comment]
merged via 7643d979c743728752f76ca7c00a7bdba9bbc61a

**Pull request closed without merging**